nexuse2e-core/src/main/java/org/nexuse2e/service/mail/SmtpSender.java

```diff
@@ -280,7 +277,10 @@ public void fillParameterMap(Map<String, ParameterDescriptor> parameterMap) {
         parameterMap.put(ENCRYPTION_PARAM_NAME, new ParameterDescriptor(ParameterType.LIST, "Encryption", "Connection" +
                 " encryption type", encryptionTypeDrowdown));
         parameterMap.put(SUBJECT_PARAM_NAME, new ParameterDescriptor(ParameterType.STRING, "Subject",
-                "E-Mail subject. Use ${systemProperty} e.g. ${nexus.message.message} for replacements. If empty, messageId will be used.",
+                "Optional E-Mail subject. Use ${systemProperty} e.g. ${nexus.message.message} for replacements. If empty, messageId will be used.",
+                ""));
+        parameterMap.put(BODY_PARAM_NAME, new ParameterDescriptor(ParameterType.TEXT, "Body",
+                "Optional E-Mail html body. Use ${systemProperty} e.g. ${nexus.message.message} for replacements.",
```

Comment:
Since it is optional, should the comments also mention what happens if the param is empty

Reply:
The description only allows one line...and there could actually be another bodypart filling the body...